src/openforms/js/components/admin/forms/VariableSelection.js

```diff
@@ -24,26 +25,51 @@ const VariableSelection = ({
   });
 
   const allFormVariables = (includeStaticVariables ? staticVariables : []).concat(formVariables);
+
+  const getVariableSource = variable => {
+    if (variable.source === VARIABLE_SOURCES.userDefined) {
+      return VARIABLE_SOURCES_GROUP_LABELS.userDefined;
+    }
+    if (variable.source === VARIABLE_SOURCES.component) {
+      return VARIABLE_SOURCES_GROUP_LABELS.component;
+    }
+    return VARIABLE_SOURCES_GROUP_LABELS.static;
+  };
```

Comment:
it looks like this could be a `switch`? I'm also not seeing a reason why this should be defined inside the component

Reply:
Yeah, you're right.. I've moved it to the `utils.js`